pkg/registry/core/pod/storage/eviction.go

```diff
@@ -103,6 +103,12 @@ func (r *EvictionREST) Destroy() {
 	// we don't destroy it here explicitly.
 }
 
+var _ rest.SingularNameProvider = &EvictionREST{}
+
+func (r *EvictionREST) GetSingularName() string {
```

Comment:
This one is the most exceptional one. It is subresource for pod but kind is set to `Eviction`. Should the singular name be `pod` or `eviction`?